binderhub/app.py

```diff
@@ -375,6 +375,22 @@ def docker_build_host_validate(self, proposal):
             raise TraitError("Only unix domain sockets on same node are supported for build_docker_host")
         return proposal.value
 
+    build_docker_config = Dict(
```

Comment:
Hello from the distant future! I'm reviewing the BinderHub config properties as part of https://github.com/jupyterhub/binderhub/pull/1521 and as far as I can tell this property isn't used. It's passed to `builder.py`, and referenced in a conditional, but its value is never used. Instead the helm chart references `.Values.config.BinderHub.buildDockerConfig` in `helm-chart/binderhub/templates/_helpers.tpl`.

Can you remember what's happening here?